docs/assignment1.md

```diff
@@ -1,62 +1,80 @@
+| F7  | Stats | The player has two main stats, `health` and `strength`, which they will use for combat. Naturally, `health` is an indicator of the player's remaining vitality (reaching 0 health will result in losing the game), whereas `strength` determines whether or not the player can defeat an enemy in combat, and how much damage they take during the battle.  |
+| F8  | Combat | Whenever a player engages in combat, their strength is compared to that of the enemy combatant. The outcome of the fight is determined by the difference in strength between both the player and their enemy:</br> - player strength < enemy strength: enemy survives, player loses a significant chunk of health </br> - player strength >= enemy strength: enemy defeated, player loses some health</br> - player strength >> enemy strength: enemy defeated, player loses no health |
+| F9  | Gamestate | The player may save or load the game at any moment using the `save` and `load` commands respectively. A savefile will contain all actions as performed by the player up until the point of saving, along with the appropriate initial game configuration. Whenever a savefile is loaded into the game, all actions are replayed so the user may proceed where they left off when saving. The player may quit the game after saving or loading, using the `quit` command at any time.  |
 
 ### Quality requirements
-Author(s): `name of the team member(s) responsible for this section`
-
-As a preamble to the table, you can discuss the main line of reasoning you followed for coming up with the quality requirements listed below.
-
-| ID  | Short name  | Quality attribute | Description  |
-|---|---|---|---|
-| QR1  | Commands sanity checks | Reliability  | When the player issues a command, the syntax of the command shall always get validated against the format specified in F2 |
-| QR2  | Extensible world | Maintainability  | The video game shall be easilty extendable in terms of levels, worlds, interaction points  |
-| QR3  | Instantaneous results | Responsiveness  | Once the scores of all soccer players are provided by the user, the results of the virtual matches shall be available within 1 second |
-| QR4  | ... | ... | ... |
-
-Each quality requirement must be tagged with the corresponding quality attribute (see corresponding slides of the first lecture for knowing them).
-
-Maximum number of words for this section: 1000
+Author(s): `Jim Cuijpers, Ajay Hitendra Mota, Anthony Wilkes`
+
+| ID  | Short name  | Quality attribute | Description  | Rationale |
+|---|---|---|---|---|
+| QR1 | JSON Validation | Reliability | The JSON files that describe the game world will be read by the engine as the game is started, and the first missing - but required - attribute shall be indicated to the user, and the game will exit. | This ensures that people who wish to modify the game files will be able to identify their mistakes easily. |
```

Comment:
What do you mean by first missing attribute? Is the validation of the content of the attribute also going to be done?